README.md

```diff
@@ -15,7 +15,7 @@ The project is currently under active development. Please report any issues you
 
 ## Usage
 
-You need Docker to run the LIFE FHIR Store. The most convenient way is to check out this repository and use the `docker-compose.yml` to bring the LIFE FHIR Store up. The default memory requirements are 4 GB. You also need to have port 8080 free on your host.
```

Comment:
I would like to keep default here, because you can change the memory settings and run it with mich less.

Reply:
No, you cannot run it with less. We have tried to run it with 2 GB and it did not work at all.

Reply:
But you can lower the memory settings of Datomic and the store before running Docker Compose.

Reply:
But then we should document how to change the memory settings, properly. Otherwise, it is fairly easy to get the store up without being able to import anything.

Reply:
What about: "The default minimum memory requirements are 4 GB. You have to change the memory settings if you have less memory available."? 

Later we can describe how to change them.

Reply:
And it really works with 2 GB Docker VM. I have tested it.

Reply:
It did not work on my machine. Do you remember? I set the available memory of Docker to 2 GB and did not change anything for datomic.

Reply:
What about: "The default minimum memory requirements are 4 GB. You have to change the memory settings for Docker and Datomic if you have less memory available."?

Reply:
...settings for the LIFE FHIR Store and Datomic...